pkg/apiserver/profiling/module.go

```diff
@@ -1,5 +1,28 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 package profiling
 
-import "go.uber.org/fx"
+import (
+	"go.uber.org/fx"
+
+	"github.com/pingcap/tidb-dashboard/pkg/apiserver/profiling/fetcher"
+)
 
-var Module = fx.Options(newFetchers, newService)
+var Module = fx.Options(
+	fx.Provide(
+		fetcher.NewClientMap,
```

Comment:
This leaks the internal `clientMap` to the outside (which can be accessed by other modules now). This doesn't seem to be good.